torch/quantization/_numeric_suite.py

```diff
@@ -227,6 +249,7 @@ def compare_model_stub(float_model, q_model, module_swap_list, data, Logger=Logg
     Args:
         float_module: the float module used to generate the q_module
         q_module: the quantized module
+        data: input data
```

Comment:
Lets make the comments more descriptive
 module_swap_list: List of float module types at which shadow modules are attached. Shadow modules log the original
floating point module output along with the output of a quantized module for the same input.